scripts/test-acceptance.sh

```diff
@@ -53,4 +53,4 @@ if [[ "${test_name}" = "accessibility" ]]; then
 fi
 
 echo "Running acceptance tests from ${test_name}.py against the sandbox..."
-nosetests ../acceptance/${test_name}.py --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
+nosetests ../acceptance/${test_name}.py:MultipleOpenAssessmentTest --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
```

Comment:
@efischer19 Yes, I know I had to drop the `scripts/test-acceptance.sh` change. I just looked at your PR and I thought this is faster. And I didn't knew that we have to write both acceptance and accessibility tests ?

For every acceptance, do we need accessibility test ?